gitlab/tests/objects/test_packages.py

```diff
@@ -117,3 +177,11 @@ def test_get_project_package(project, resp_get_package):
 def test_delete_project_package(project, resp_delete_package):
     package = project.packages.get(1, lazy=True)
     package.delete()
+
+
+def test_list_project_packages(project, resp_list_package_files):
```

Comment:
Just a small one here so it doesn't clash with the test name above:
 
```suggestion
def test_list_project_package_files(project, resp_list_package_files):
```

Reply:
Ah, that was an oversight when I had copied my test back into `test_packages.py`. I see you added the change to the PR - is anything else required?

Reply:
Sorry, forgot to merge when I was waiting for the tests ;) will merge now, thanks for the ping!

Reply:
Thanks!